ipaserver/install/krbinstance.py

```diff
@@ -45,6 +45,7 @@
 from ipaserver.install import ldapupdate
 
 from ipaserver.install import certs
+from ipaserver.masters import find_providing_servers
```

Comment:
The 4.7 branch doesn't have the ``ipaserver.masters` module. The function is located in ``ipaserver.install.service``.

Reply:
Thanks for the hint, PR updated.